test/function_tests/backplane_operator_install_test/backplane_operator_install_test.go

```diff
@@ -198,6 +203,58 @@ var _ = Describe("BackplaneConfig Test Suite", func() {
 				})
 			}
 		})
+		It("Should check that the config spec has propagated", func() {
+
+			tests := []struct {
+				Name           string
+				NamespacedName types.NamespacedName
+				ResourceType   client.Object
+			}{
+				{
+					Name:           "OCM Webhook",
+					NamespacedName: types.NamespacedName{Name: "ocm-webhook", Namespace: BackplaneOperatorNamespace},
+					ResourceType:   &appsv1.Deployment{},
+				},
+				{
+					Name:           "OCM Controller",
+					NamespacedName: types.NamespacedName{Name: "ocm-controller", Namespace: BackplaneOperatorNamespace},
+					ResourceType:   &appsv1.Deployment{},
+				},
+				{
+					Name:           "OCM Proxy Server",
+					NamespacedName: types.NamespacedName{Name: "ocm-proxyserver", Namespace: BackplaneOperatorNamespace},
+					ResourceType:   &appsv1.Deployment{},
+				},
+				{
+					Name:           "Cluster Manager Deployment",
+					NamespacedName: types.NamespacedName{Name: "cluster-manager", Namespace: BackplaneOperatorNamespace},
+					ResourceType:   &appsv1.Deployment{},
+				},
+				{
+					Name:           "Hive Operator Deployment",
+					NamespacedName: types.NamespacedName{Name: "hive-operator", Namespace: BackplaneOperatorNamespace},
+					ResourceType:   &appsv1.Deployment{},
+				},
+			}
+
+			By("Ensuring the spec is correct")
+			for _, test := range tests {
+
+				Eventually(func() bool {
+					// component := &unstructured.Unstructured{}
+					err := k8sClient.Get(ctx, test.NamespacedName, test.ResourceType)
+					if err != nil {
+						fmt.Fprintf(GinkgoWriter, "could not get component %s\n", test.Name)
+					}
+
+					componentSelector := test.ResourceType.(*appsv1.Deployment).Spec.Template.Spec.NodeSelector
+
+					return reflect.DeepEqual(componentSelector, backplaneNodeSelector)
+
+				}, installTimeout, interval).Should(BeTrue())
```

Comment:
The installTimeout is a 5 minutes timeout meant to be the max limit for how long it could take to completely install. For checking that the deployment spec is updated this seems overkill. I would think 30 seconds is more than enough to validate that.